### ISSUE TITLE
Can a browse category show the facets?

### ISSUE THREAD

**Issue by mejackreed** [sul-dlss/dlme#1107]
Brought up by @mwerla on 2021-01-13 and mentioning the usefulness to be able to refine predefined browse categories by using facets

**Comment by corylown** [comment]
I _think_ this issue is requesting that facets appear on pages like this one: https://dlmenetwork.org/library/browse/group/manuscripts/persian-manuscripts.
<img width="1102" alt="Screen Shot 2022-10-27 at 3 57 36 PM" src="https://user-images.githubusercontent.com/458247/198386192-e43c7909-0041-407d-883f-30fb6f3169f5.png">

**Comment by ggeisler** [comment]
Yes, the ask is for a facet sidebar on the browse results page. This would require new design and development because Spotlight browse pages are intentionally sidebar- and facet-less. 

There should be a in-depth discussion at the appropriate time about whether to do this, because I think it would blur the lines between search and browse functionality, though I understand the reason it is being asked for in DLME, given the large number of items DLME browse categories can contain.

**Comment by caaster** [comment]
Simply another data point: this functionality has never been requested by a Spotlight at Stanford exhibit creator/user.
